specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/virtualMachineScaleSet.json

```diff
@@ -3489,6 +3489,79 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/virtualmachines/{instanceId}/attachDetachDataDisks": {
+      "post": {
```

Comment:
Since this is an action on VM of VM ScaleSet (uniform) similar to start, restart, reimage, redeploy, etc, following POST HTTP verb here. Please suggest.

Refer existing action APIs: https://learn.microsoft.com/en-us/rest/api/compute/virtual-machine-scale-set-vms/start?tabs=HTTP

Reply:
ok understand. If the older design, adheres to this REST semantics, we are good here.